network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -442,22 +442,27 @@ class Meta:
         verbose_name = "Buyers Guide Page"
 
 
-class BuyersGuidePageHeroSupportingArticleRelation(TranslatableMixin, Orderable):
+class BuyersGuidePageHeroSupportingPageRelation(TranslatableMixin, Orderable):
     page = cluster_fields.ParentalKey(
         "wagtailpages.BuyersGuidePage",
-        related_name="hero_supporting_article_relations",
+        related_name="hero_supporting_page_relations",
     )
-    article = models.ForeignKey(
-        "wagtailpages.BuyersGuideArticlePage",
+    supporting_page = models.ForeignKey(
+        "wagtailcore.Page",
```

Comment:
Using the [wagtail docs](https://docs.wagtail.org/en/stable/reference/pages/panels.html#pagechooserpanel) for reference, I updated the FK to point to `wagtailcore.Page `as this will allow us to select from more than one page type. 

**Note:** Existing choices in this field remain even after running migrations

We then limit the choices to `BuyersGuideArticlePage` and `BuyersGuideCampaignPage` in this fields `PageChooserPanel` definition:

https://github.com/MozillaFoundation/foundation.mozilla.org/blob/4f75c5fd496d56e533157b38061ff08e07a03135/network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py#L458-L461